src/p2p/net_node.inl

```diff
@@ -2822,17 +2819,12 @@ namespace nodetool
       address = epee::string_tools::get_ip_string_from_int32(ipv4.ip());
       port = epee::string_tools::num_to_string_fast(ipv4.port());
     }
-    else if (is_ipv6)
+    else
     {
       const epee::net_utils::ipv6_network_address &ipv6 = na.as<const epee::net_utils::ipv6_network_address>();
```

Comment:
This throws bad_cast if a non-ipv6 address is passed in, don't think any of the code using this expects that

Reply:
Oh I see the check and assert above.